src/ToricDivisors.jl

```diff
@@ -90,6 +150,11 @@ end
 export is_very_ample
 
 
+"""
+    is_numerically_effective( d::toric_divisor )
```

Comment:
Is it not common to abbreviate this to `nef` these days?

Reply:
Thx. Just changed it.